core/src/impl/Kokkos_ExecPolicy.cpp

```diff
@@ -18,9 +18,11 @@
 #define KOKKOS_IMPL_PUBLIC_INCLUDE
 #endif
 
-#include <Kokkos_Core.hpp>
+#include <iostream>
```

Comment:
Why did you add `<iostream>` and move `<Kokkos_Core.hpp>`?

Reply:
Most of the includes related changes have been moved to #6482.